database/postgres/postgres.go

```diff
@@ -29,7 +29,10 @@ const (
 var log = loggo.GetLogger("postgres")
 
 func init() {
-	loggo.ConfigureLoggers(logLevel)
+	err := loggo.ConfigureLoggers(logLevel)
+	if err != nil {
+		fmt.Println(err)
```

Comment:
Should this be a panic, to match all other init functions?

Reply:
Yeah that's fine but I used the surrounding code as the guide on what to do there.